packages/functions/e2e/functions.e2e.js

```diff
@@ -51,6 +51,24 @@ describe('functions()', function () {
       const response = await functionRunner();
       response.data.should.equal(region);
     });
+
+    it('accepts passing in a custom url string as first arg to an app', async function () {
+      const customUrl = 'https://us-central1-react-native-firebase-testing.cloudfunctions.net';
+      const functionsForRegion = firebase.app().functions(customUrl);
+
+      functionsForRegion._customUrlOrRegion.should.equal(customUrl);
+      functionsForRegion.app.should.equal(firebase.app());
+      functionsForRegion.app.name.should.equal(firebase.app().name);
+
+      firebase.app().functions(customUrl).app.should.equal(firebase.app());
```

Comment:
Can we use the instance we created above for this, and also the one below?

Reply:
Updated 👍

Reply:
Updated 👍

Reply:
verified